<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weatherItem</class>
 <widget class="QWidget" name="weatherItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>376</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>662</width>
    <height>376</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>662</width>
    <height>376</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSoft">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(157, 133, 255,0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="reg.qrc">
          <normaloff>:/weather/weatherImages/search.png</normaloff>:/weather/weatherImages/search.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelData">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Arial&quot;;
background-color: rgba(255, 255, 255,0);</string>
        </property>
        <property name="text">
         <string>2023/11/23 星期四</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPix">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="reg.qrc">:/weather/weatherImages/type/ZhenYu.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="labelCity">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(255, 255, 255,0);</string>
           </property>
           <property name="text">
            <string>北京</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelWether">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255,0);
font: 12pt &quot;微软雅黑&quot;;
</string>
           </property>
           <property name="text">
            <string>多云转晴 </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labeTem">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255,0);
font: 12pt &quot;微软雅黑&quot;;
</string>
           </property>
           <property name="text">
            <string>20~23</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelHightTem">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 50pt &quot;Arial&quot;;
background-color: rgba(0, 255, 255, 0);
</string>
           </property>
           <property name="text">
            <string>32</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="labelWeather">
           <property name="text">
            <string>晴</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labeldate">
           <property name="text">
            <string>25</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>日期</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>空气质量</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>风级</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6" colspan="2">
          <widget class="QLabel" name="labelHight">
           <property name="text">
            <string>22</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelAir">
           <property name="text">
            <string>良</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>最高温</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QLabel" name="labelWind">
           <property name="text">
            <string>3级</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" colspan="2">
          <widget class="QLabel" name="labelLow">
           <property name="text">
            <string>-9</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>天气图</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>最低温</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="reg.qrc"/>
 </resources>
 <connections/>
</ui>
